src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceUtils.kt

```diff
@@ -210,10 +210,14 @@ inline fun <R> whenNetworkServiceObject(
     isEvChargingUnit: (EvChargingUnit) -> R,
     isTapChangerControl: (TapChangerControl) -> R,
     isSeriesCompensator: (SeriesCompensator) -> R,
+    isSynchronousMachine: (SynchronousMachine) -> R,
```

Comment:
todo: Add all new parameters to the end of the function to avoid headaches for users of the SDK.

Reply:
moved and docstring added